crypto/md5/md5_local.h

```diff
@@ -15,7 +15,8 @@
 #ifdef MD5_ASM
 # if defined(__i386) || defined(__i386__) || defined(_M_IX86) || \
      defined(__x86_64) || defined(__x86_64__) || defined(_M_AMD64) || \
-     defined(_M_X64) || defined(__aarch64__)
+     defined(_M_X64) || defined(__aarch64__) || defined(__loongarch__) || \
+     defined(__loongarch64)
```

Comment:
```suggestion
     defined(_M_X64) || defined(__aarch64__) || (defined(__loongarch__) && __loongarch_grlen == 64)
```

If I read the code correctly, it requires 64-bit GPR but does not require LP64 ABI (i.e. it should works with ILP32 ABI too).  If it requires LP64 (LP64S/LP64F/LP64D) ABI, just test `defined(__loongarch_lp64)`.

`__loongarch64` is obsolete because it actively confuses people to mix up "LP64 ABI" and "64-bit GPR".  These are two different concepts.  64-bit GPR is a hardware feature, but LP64 ABI is a software interface.

Reply:
Yes. Thanks for your review.  Using ```__loongarch_grlen``` is more appropriate.